Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -227,6 +238,16 @@ final class RewardsCollectionViewController: UICollectionViewController {
       ?|> \.isActive .~ !isHidden
   }
 
+  private func goToAddOnSelection(
```

Comment:
Left these args the same as `goToRewards` for now, they will likely be a bit different but we'll figure that out in the next PR.